lib/metadata/linux/LinuxUtils.rb

```diff
@@ -115,5 +115,29 @@ def self.parse_chkconfig_list(lines)
 
       return ret
     end
+
+    def self.parse_systemctl_list(lines)
+      ret = []
+      return ret if lines.nil? || lines.empty?
```

Comment:
Gah. There is next inside, so I need to call compact at the end, so it removes all the nils caused by 'next'. So it's another unnecessary pass through the array, but if you think the code is more concise, then let it be.